optaplanner-core/src/build/findbugs-excludes.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+  <Match>
+    <!--
+    Uninitialized fields are intentional in this class - to make sure the uninitialized value is the same as JVM
+    would use for uninitialized fields in facts.
+    -->
+    <Class name="org.optaplanner.core.impl.score.director.drools.testgen.fact.TestGenPrimitiveValueProvider"/>
```

Comment:
Can we do this with annotations instead of an external xml file?

Reply:
it seems there is, but the question is if it introduces an unwelcome dependency (if we make it optional=true, I think it might be ok, if that works for users of course...)
https://stackoverflow.com/a/1829960/472109

Reply:
I think `provided` is what we want - it will be available in compile time but not required for runtime.

Reply:
Either way, I am ok to go with an excludes file for now too.
But let's revert the listFiles() changes.

Reply:
I like the annotation approach, the justification is there right with the code, much more intuitive. The listFiles() won't be reverted - see the other comment.